packages/plugins/plugin-preview/src/stories/testing.tsx

```
@@ -20,16 +20,16 @@ export type DefaultStoryProps<T extends Obj.Any> = {
   const object = useMemo(() => createObject(), [createObject]);
-  const roles: SurfaceComponentProps['role'][] = ['card--popover', 'card--intrinsic'];
+  const roles: SurfaceComponentProps['role'][] = ['card--intrinsic', 'card--popover'];
 
   return (
-    <div className='grid grid-rows-4 gap-16 plb-8'>
+    <div className='bs-full grid grid-rows-2 p-16 gap-16'>
       {roles.map((role, i) => (
-        <div key={i} className='flex justify-center'>
+        <div key={i} className='flex bs-full justify-center overflow-hidden'>
           <div className='flex flex-col gap-4 is-full items-center'>
             <label className='text-sm text-description'>{role}</label>
```

Biome (2.3.14)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)